includes/class-sensei-lesson.php

```diff
@@ -287,8 +290,23 @@ public function lesson_prerequisite_meta_box_content() {
 			'suppress_filters' => 0,
 			'post_status'      => [ 'publish', 'draft' ],
 		);
+
+		if ( $current_lesson_course ) {
+			// Meta query args to only get Lesson Posts from current Course.
+			$meta_query_args = array(
```

Comment:
We could just assign this array directly to `$post_args['meta_query']` and eliminate the need for this variable.

Reply:
Done!